Eastern-Research-Group/mywaterway
# Feature/379 cyan panel edits

## Related Issues:
* [HMW-379](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-379)

## Main Changes:
* Removed the _Elevation_, _Centroid Latitude_, _Centroid Longitude_, and _Count_ rows from the CyAN panel.
* Added tooltips to several rows.
* Updated the _Current Water Conditions_ descriptive text.
* Added descriptive text to the CyAN panel.
* Added a histogram to to the CyAN panel to display daily cell concentration data.

## Steps To Test:
1. Navigate to the Lake Mattamuskeet watershed, i.e. http://localhost:3000/community/030201050101/monitoring, then expand the Lake Mattamuskeet accordion item.
2. Confirm that the appearance of the panel aligns with the edits requested (attached in the corresponding Jira ticket).
